### PR TITLE
finished typescript-primitives-and-variables, installed typescript

### PR DESCRIPTION
**Pull request by brendan-kormos**
image

### CODE REVIEW

File: typescript-primitives-and-variables/main.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+console.log('iscool typeof', typeof isCool);
+console.log('totalpets ', totalPets);
+console.log('totalpets typeof', typeof totalPets);
+
+const unknown: null = null;
+
+console.log('unknown ', unknown);
+console.log('unknown typeof', typeof unknown);
+
+const never: undefined = undefined;
```
Comment (robgmerrill) [inline typescript-primitives-and-variables/main.ts:16]: let never: undefined;

**Review by robgmerrill — CHANGES_REQUESTED** [review]
One quick change requested

File: typescript-primitives-and-variables/main.ts (hunk 1 above)
Comment (brendan-kormos) [reply typescript-primitives-and-variables/main.ts:16]: I thought I heard you were saying something about that but the zoom was lagging and screenshare wouldn't load for me.

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**